content/guides/output-management.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Output Management
```

Comment:
Or... how about the one from #949 -- __Managing Built Files__ or __Managing Output Files__?

I think one of those would be an improvement without locking it in to an html-only discussion.